### ISSUE TITLE
New language: Macedonian

### ISSUE THREAD

**Issue by ZoranPandovski** [squidfunk/mkdocs-material#3822]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translation

```jinja
{% macro t(key) %}{{ {
  "language": "mk",
  "direction": "насока",
  "clipboard.copy": "Копирај во таблата",
  "clipboard.copied": "Копирано",
  "edit.link.title": "Уредете ја оваа страница",
  "footer.previous": "Претходно",
  "footer.next": "Следно",
  "footer.title": "Подножје",
  "header.title": "Заглавје",
  "meta.comments": "Коментари",
  "meta.source": "Извор",
  "nav.title": "Наслов за навигација",
  "search.config.lang": "enmk,
  "search.config.pipeline": "trimmer, stopWordFilter",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "Пребарување",
  "search.reset": "Чисти",
  "search.result.initializer": "Иницијализирање на пребарувањето",
  "search.result.placeholder": "Напишете за да започнете со пребарување",
  "search.result.none": "Нема соодветни документи",
  "search.result.one": "1 документ што се совпаѓа",
  "search.result.other": "# соодветни документи",
  "search.result.more.one": "Уште 1 на оваа страница",
  "search.result.more.other": "Уште # на оваа страница",
  "search.result.term.missing": "Недостасува",
  "select.language.title": "Изберете јазик",
  "select.version.title": "Изберете верзија",
  "skip.link.title": "Прескокнете до содржината",
  "source.link.title": "Одете до складиштето",
  "source.file.date.updated": "Последно ажурирање",
  "source.file.date.created": "Создаден",
  "tabs.title": "Јазичиња",
  "toc.title": "Содржина",
  "top.title": "Вратете се на почетокот"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for providing the translations!

**Comment by squidfunk** [comment]
Added in e77aa0015.

**Comment by squidfunk** [comment]
Released as part of 8.2.10.
